user/cinnamon/template.py

```diff
@@ -0,0 +1,55 @@
+pkgname = "cinnamon"
+pkgver = "6.2.9"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["gobject-introspection", "intltool", "meson", "pkgconf"]
+makedepends = [
+    "at-spi2-core-devel",
+    "cinnamon-menus-devel",
+    "cjs-devel",
+    "dbus-devel",
+    "glib-devel",
+    "gstreamer-devel",
+    "gtk+3-devel",
+    "libx11-devel",
+    "libxml2-devel",
+    "mesa-devel",
+    "muffin-devel",
+    "networkmanager-devel",
+    "polkit-devel",
+    "xapp-devel",
+]
+# depends on cinnamon-session, for later
+depends = [
+    "caribou",
+    "cinnamon-screensaver",
+    "cinnamon-session",
+    "nemo",
+    "network-manager-applet",
+]
+pkgdesc = "Linux desktop that provides a traditional user experience"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://projects.linuxmint.com/cinnamon"
+source = (
+    f"https://github.com/linuxmint/cinnamon/archive/refs/tags/{pkgver}.tar.gz"
+)
+sha256 = "a0b15b98a899532d531689ea11ff6805a87fcf5994e238322b6c916bf8c29919"
+# meson.build: tests are not currently functional
+options = ["!check"]
+
+
+def post_install(self):
+    self.install_dir("etc/xdg/menus/applications-merged", empty=True)
```

Comment:
what happens if this directory does not exist?

Reply:
cbuild will output an "unknown provider" error when scanning for symlink dependencies.

Reply:
That directory doesn't need to be present, because by default it scans for .desktop files in /usr/share/applications.